Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -142,11 +159,12 @@ export const projectDispatcher = () => {
     (callbackHelpers: CallbackInterface) => async (newProjectData: any) => {
       const { set, snapshot } = callbackHelpers;
       const dispatcher = await snapshot.getPromise(dispatcherState);
+      const rootBotProjectId = await snapshot.getPromise(rootBotProjectIdSelector);
       try {
-        const { templateId, name, description, location, schemaUrl, locale, qnaKbUrls } = newProjectData;
```

Comment:
the qnaKbUrls were passed in to import a url for a qnamaker bot in the creation wizard. Is this scenario no longer valid?

Reply:
@srinaath Yes, it's out of date after 'QnA Refinement' PR been merged